src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
Hi @pedro-ruas. 

See the class in platform  "HTML+CSS Advanced/BEM > Mix Block with parent element", that recommended this class in html and body tags. Please confirm if there is wrong content in this class provided by platform. 

In chat br_fs_feb24_qna exist a more detailed comments about this. Kind reminder for resolve this topic in chat before reviews, to avoid unecessary pull requests.